app/stac_api/models.py

```diff
@@ -566,7 +568,11 @@ class ItemLink(Link):
     )
 
 
-ASSET_KEEP_ORIGINAL_FIELDS = ["eo_gsd", "geoadmin_variant", "proj_epsg"]
+ASSET_KEEP_ORIGINAL_FIELDS = ["name"] + UPDATE_SUMMARIES_FIELDS
+
+
+def upload_to_asset_path(instance, filename):
```

Comment:
`get_upload_to_asset_path` might be more appropriate here, this function don't upload but return a path string.

Reply:
jup, changed it.